src/bikeracks/votes.py

```diff
@@ -51,120 +47,97 @@ def get_vote_status():
     if result:
         result = h.dict_from_row(result)
         return jsonify(result)
```

Comment:
If this `return` is indented under the `if` on line 47, then you'll return `None` when that `if` fails, which is a flask error (you must return a response).

What is a sensible thing to do if the client asks for status for a rack that doesn't exist? Or a user that doesn't exist? Or a rack that the user has never voted on? Bad request? Not found? I dunno. Not 500 though.

```python
if not result:
  return "User vote not found", 404
```

Reply:
I unindented the return statement. Would that be good enough?

Reply:
What happens if you call `jsonify(None)`?